pkg/ui/src/views/cluster/containers/nodeGraphs/index.tsx

```diff
@@ -132,14 +135,10 @@ export class NodeGraphs extends React.Component<NodeGraphsProps, {}> {
     },
   );
 
-  refresh(props = this.props) {
-    if (!props.nodesQueryValid) {
-      props.refreshNodes();
-    }
-    if (!props.livenessQueryValid) {
-      props.refreshLiveness();
-    }
-  }
+  refresh = () => {
```

Comment:
`nodesQueryValid` and `livenessQueryValid` validations are performed in `CachedDataReducer` when call refresh for any resource.